frontend/src/pages/index.js

```diff
@@ -17,59 +17,69 @@ import {
   Row,
   Badge
 } from 'reactstrap'
+import { connect } from 'react-redux'
+import { fetchCandidates, addFilter, removeFilter } from '../actions'
+import { bindActionCreators } from 'redux'
+import { yearsenum, statusenum, rolesenum } from '../utils/enums'
 
 type Props = {
-  result: {}
+  candidates: Array<any>,
+  loading: boolean,
+  error: boolean,
+  filters: Object,
+  sort: Object
+}
+
+const mapDispatchToProps = dispatch => {
+  return bindActionCreators(
+    {
+      fetchCandidates,
+      addFilter,
+      removeFilter
+    },
+    dispatch
+  )
+}
+
+const mapStateToProps = state => {
+  return {
+    candidates: state.candidateListPage.candidates,
+    loading: state.candidateListPage.candidatesLoading,
+    error: state.candidateListPage.candidatesError,
+    filters: state.candidateListPage.filters,
+    sort: state.candidateListPage.sort
+  }
```

Comment:
cool thing you could do is wrap 45-51 in `( )` instead of `{ }` and you can remove the return statement.

so..
```Javascript
const asdfasdf = state => ({text:"asdf"})
```
up to you though